shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Should it be 2018, though?  I don't see anything other than 2013 in surrounding files.  Also, I think changing it to 2018 is considered a new license which then breaks CI on the license file because the total license count changes.